src/test/selftest-parallel

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
```

Comment:
we may want to add
`set -e` here so that the script fails if any command fails

Reply:
I was thinking to use a handler for killing children, but not needed at this point I agree.